From 330b973824c6d835fd8c43f56d2bc05f931fc919 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Fri, 19 Jul 2019 17:49:00 +0100
Subject: [PATCH 720/773] drm/vc4: Ensure zpos is always initialised

The compiler is warning that default_zpos can be used
uninitialised as there is no default case to catch all plane
types.
No other plane types should ever be presented to vc4_fkms_plane_init,
but add a default case regardless.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/gpu/drm/vc4/vc4_firmware_kms.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -773,6 +773,7 @@ static struct drm_plane *vc4_fkms_plane_
 	 * other layers as requested by KMS.
 	 */
 	switch (type) {
+	default:
 	case DRM_PLANE_TYPE_PRIMARY:
 		default_zpos = 0;
 		break;
